metadata.rb

```diff
@@ -10,14 +10,8 @@
 
 depends 'yum-epel'
 
-supports 'debian'
-supports 'ubuntu'
-supports 'centos'
+supports 'amazon', '>= 2'
+supports 'centos', '>= 7'
+supports 'debian', '>= 9'
 supports 'redhat'
-supports 'scientific'
```

Comment:
scientific is just RHEL 7 so this would continue to work

Reply:
TIL!